lib/browser/desktop-capturer.js

```diff
@@ -10,27 +10,33 @@ const deepEqual = (a, b) => JSON.stringify(a) === JSON.stringify(b)
 // A queue for holding all requests from renderer process.
 let requestsQueue = []
 
-const electronSources = 'ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES'
-const capturerResult = (id) => `ELECTRON_RENDERER_DESKTOP_CAPTURER_RESULT_${id}`
+function defer () {
```

Comment:
I'm not a huge fan of this "deferred" pattern, is there any alternative?

Reply:
Me neither

Reply:
@MarshallOfSound I've removed the `deferred`, but I still need to store the `resolve` function to keep it working without having to refactor the C++ code.